e2e/qr-code-reader.e2e.ts

```diff
@@ -55,7 +62,10 @@ test("Read QR throws error because camera stream fails", async () => {
 
   const page = await context.newPage();
 
-  await testQRCode({ page });
+  await testQRCode({
+    page,
+    screenshotName: "Read-QR-throws-error.png",
```

Comment:
Is the screenshot there just to verify that the toast appeared? Maybe just check that the toast is there directly rather than through a screenshot?

Reply:
The test is there to test that an error was throw. We cannot test the QR code in a NodeJS context because the library requires a rendering window.

Reply:
Can you test that the toast is in the DOM and contains an error message?